src/main/resources/db/migration/common/V2025.03.18.0001__fix_null_event_context.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM timeline_event_contextual_info
```

Comment:
Good shout; yes, we defo want to do this 👍 

On the assumption that we move the notes field into the conditional block (my other comment), what do you think about also deleting the `COMPLETED_INDUCTION_NOTES` record where the value is empty string?

I know we support no/empty notes; but at the moment where the feature toggle is not enabled, there shoudnt be any records set this way anyway 🤷